packages/service-client/odsp-client/src/index.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export {
+	OdspConnectionConfig,
+	OdspClientProps,
+	OdspContainerServices,
+	OdspServiceAttributes,
+	IOdspAudience,
+	OdspMember,
+} from "./interfaces";
+export { OdspClient } from "./OdspClient";
+export { OdspAudience } from "./OdspAudience";
```

Comment:
Do we need to export the class, or is just the interface sufficient? Do external users need to construct these?